core/include/kernel/abort.h

```diff
@@ -27,11 +27,15 @@ struct abort_info {
 
 /* Print abort info to the console */
 void abort_print(struct abort_info *ai);
+
 /* Print abort info + stack dump to the console */
 void abort_print_error(struct abort_info *ai);
 
 void abort_handler(uint32_t abort_type, struct thread_abort_regs *regs);
 
+/* Platform overload, should be implemented in platform code */
```

Comment:
Could you explicitly mention `CFG_EXTERNAL_ABORT_PLAT_HANDLER` must be enabled to have the platform handler be called.  I think the function label should explicitly mention `_external_abort_` if called only for external aborts.